src/components/forms/InputForField.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { fieldSetChildStyle } from './fieldSetChildStyle';
+import { textChangeHandler } from "./textChangeHandler";
+
+export const InputForField = 
```

Comment:
this is a generic input that knows how to read and update a "field", which is a new construct i introduced in this PR, to dry up some repetitive code for setting / getting values within the state.